docs/source/pythonapi/examples/tally-arithmetic.ipynb

```diff
@@ -0,0 +1,1512 @@
+       "      <th></th>\n",
+       "      <th></th>\n",
+       "      <th></th>\n",
+       "      <th></th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>total</td>\n",
+       "      <td>fission</td>\n",
+       "      <td>1.080124</td>\n",
+       "      <td>0.008126</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "    nuclide    score      mean  std. dev.\n",
+       "bin                                      \n",
+       "0     total  fission  1.080124   0.008126"
+      ]
+     },
+     "execution_count": 28,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Compute fast fission factor factor using tally arithmetic\n",
+    "therm_fiss_rate = sp.get_tally(name='therm. fiss. rate')\n",
+    "tot_fiss_rate = sp.get_tally(name='tot. fiss. rate')\n",
+    "fast_fiss = tot_fiss_rate / therm_fiss_rate\n",
+    "fast_fiss.get_pandas_dataframe()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The thermal flux utilization is calculated as\n",
+    "$$f=\\frac{\\Sigma_{a,thermal}^F\\phi}{\\Sigma_a\\phi}$$\n",
+    "where $F$ denotes fuel."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 29,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div style=\"max-height:1000px;max-width:1500px;overflow:auto;\">\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>nuclide</th>\n",
+       "      <th>score</th>\n",
+       "      <th>mean</th>\n",
+       "      <th>std. dev.</th>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>bin</th>\n",
+       "      <th></th>\n",
+       "      <th></th>\n",
+       "      <th></th>\n",
+       "      <th></th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>total</td>\n",
+       "      <td>absorption</td>\n",
+       "      <td>0.769263</td>\n",
+       "      <td>0.004172</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "    nuclide       score      mean  std. dev.\n",
+       "bin                                         \n",
+       "0     total  absorption  0.769263   0.004172"
+      ]
+     },
+     "execution_count": 29,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Compute thermal flux utilization factor using tally arithmetic\n",
+    "fuel_therm_abs_rate = sp.get_tally(name='fuel therm. abs. rate')\n",
+    "therm_util = fuel_therm_abs_rate / abs_rate\n",
```

Comment:
This was actually a typo in my original Notebook - this should be:

```python
therm_util = fuel_therm_abs_rate / therm_abs_rate
```

Checkout @bhermanmit's nice writeup on the four factor formula on the web [here](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=4&ved=0CDUQFjADahUKEwichNDq253HAhXIKYgKHRjUB_o&url=https%3A%2F%2F09393934445762878342.googlegroups.com%2Fattach%2F4db789390b83d4c4%2F4_factor_OpenMC.pdf%3Fpart%3D5%26vt%3DANaJVrFGxsZ4qKKio8tOEHpREDsdSDuWQgIdIMEJASfcYoByiZDEdiQbzuO22lrfPRHmtI_fzfsUTnzxkBdPQAfW8Vx0eM_8po0i6AN9bsVe8Fuv_uy0sig&ei=1C3IVZzKL8jToASYqJ_QDw&usg=AFQjCNGMDvyg15vfr93LIq7p_IvGpcek2w&sig2=w4LMwerw3I0ntPBrPDEkPw&bvm=bv.99804247,d.cGU).